File: 2021-09/이상한문자만들기.js
function solution(s) {
    return s.split(" ")
    .map(str => 
         str.split("")
         .map((m, idx) => idx % 2 === 0 ? m.toUpperCase() : m.toLowerCase())
         .join("")
    )
    .join(" ");
}

console.log("try hello world"); // "TrY HeLlO WoRlD"

File: programmers/src/main/java/level1/이상한문자만들기.java
package level1;

public class 이상한문자만들기 {

	public String solution(String s) {
		StringBuilder answer = new StringBuilder();
		int cnt = 0;
		String[] array = s.split("");

		for(String ss : array) {
			cnt = ss.contains(" ") ? 0 : cnt + 1;
			answer.append(cnt % 2 == 0 ? ss.toLowerCase() : ss.toUpperCase());
		}
		return answer.toString();
	}

}
